src/crepe/api/Event.h

```diff
@@ -39,7 +40,7 @@ class KeyReleaseEvent : public Event {
 class MousePressEvent : public Event {
 public:
 	//! mouse position
-	ivec2 mouse_pos = {0, 0};
+	vec2 mouse_pos = {0, 0};
```

Comment:
Krijgen we nu de muis positie in game wereld coordinaten? Zo ja, zet dit er even bij in de doxygen comments